cdap-docs/application-templates/source/etl/templates/transformations/script.rst

```diff
@@ -6,19 +6,64 @@
 Transformations: Script 
 =======================
 
-.. rubric:: Description: Executes user-provided Javascript
+.. rubric:: Description
 
 Executes user-provided Javascript in order to transform one record into another.
+Input records are converted into JSON objects which can be directly accessed in
+Javascript. The transform expects to receive a JSON object as output, which it will
+convert back to a record in Java to pass to downstream transforms and sinks. 
 
-**Script:** Javascript defining how to transform one record into another. The script must
-implement a function called 'transform', which take as input a JSON object that represents
+.. rubric:: Use Case
+
+The Script transform is used when other transforms cannot meet your needs.
+For example, you may want to multiply a field by 1024 and rename it from 'gigabytes'
+to 'megabytes'. Or you may want to convert a timestamp into a human readable date string. 
+
+.. rubric:: Properties
+
+**script:** Javascript defining how to transform one record into another. The script must
+implement a function called 'transform', which takes as input a JSON object that represents
 the input record, and returns a JSON object that respresents the transformed input.
```

Comment:
ok i see i thought you were commenting on something else. this is a typo comment.